harness/determined/common/experimental/experiment.py

```diff
@@ -59,8 +60,18 @@ def _to_bindings(self) -> bindings.v1GetExperimentsRequestSortBy:
 class ExperimentOrderBy(enum.Enum):
     """Specifies whether a sorted list of experiments should be in ascending or
     descending order.
+
+    This class is deprecated in favor of ``OrderBy`` and will be removed in a future
+    release.
     """
 
+    warnings.warn(
+        "'ExperimentOrderBy' is deprecated and will be removed in a future "
+        "release. Please use 'OrderBy' instead.",
```

Comment:
Could you please add a module name to the `OrderBy`? Just something to help with the deprecation.